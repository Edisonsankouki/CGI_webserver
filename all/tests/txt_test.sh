cd ..
python3 webserv.py config.cfg &
PID=$!
cd -
str=$(curl localhost:8070/test.txt | diff -w - txt_test.out)
if [[ "$str" == "" ]];then
    echo test_txt passed
else
    echo test_txt failed
fi 
kill $PID
