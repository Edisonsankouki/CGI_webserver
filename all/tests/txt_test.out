<h1>BYE BYE</h1>